enclave-runtime/src/lib.rs

```diff
@@ -397,11 +408,11 @@ fn dispatch_parentchain_blocks_for_import<WorkerModeProvider: ProvideWorkerMode>
 			return Err(Error::NoParentchainAssigned)
 		};
 
-	import_dispatcher.dispatch_import(blocks_to_sync)?;
+	info!("DISPATCHING IMPORT!@#!@#!@");
+	import_dispatcher.dispatch_import(blocks_to_sync, events_to_sync)?;
 	Ok(())
 }
 
-// ANDREW
 /// Validates the events coming from the parentchain
 fn validate_events(
```

Comment:
@clangenb is there a good way you see to unit test? Perhaps an integration test is necessary?